### PR TITLE
feat: support for npm

### PR DESCRIPTION
**Pull request by conwnet**
We add support for npm in this PR.
For a preview: https://alpha.npmjs1s.com
also add google analytics code here for basic traffic info.

### CODE REVIEW

File: resources/index-dev-vscode.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		<link rel="icon" media="(prefers-color-scheme:dark)" href="/favicon-dark.svg" type="image/svg+xml" />
 		<link rel="manifest" href="/manifest.json" />
 		<title>GitHub1s</title>
-		<script>if (window.location.hostname.match(/\.?gitlab1s\.com$/i)) { window.document.title = 'GitLab1s'; } else if (window.location.hostname.match(/\.?bitbucket1s\.org$/i)) { window.document.title = 'Bitbucket1s'; };</script>
+		<script>if (window.location.hostname.match(/\.?gitlab1s\.com$/i)) { window.document.title = 'GitLab1s'; } else if (window.location.hostname.match(/\.?bitbucket1s\.org$/i)) { window.document.title = 'Bitbucket1s'; } else if (window.location.hostname.match(/\.?npmjs1s\.com$/i)) { window.document.title = 'npm1s'; };</script>
```
Comment (xcv58) [inline resources/index-dev-vscode.html:12]: I think we can refactor this to make it reusable in all `.html` files without copy paste, I will look into it.

**Review by xcv58 — APPROVED** [review]
This is awesome! lgtm!

File: resources/index-dev-vscode.html (hunk 1 above)
Comment (conwnet) [reply resources/index-dev-vscode.html:12]: yes! thank you for your advice,  i think we can migrate these to the build scripts

Comment (xcv58) [reply resources/index-dev-vscode.html:12]: I'm working on this, will share a PR soon.

**Pull request merged**